### ISSUE TITLE
Toast component aria-live have invalid "alert" option

### ISSUE THREAD

**Issue by Chityalaakhil** [ng-bootstrap/ng-bootstrap#4776]
### Bug description:
Toast component aria-live options have invalid "alert" option. 

Valid aria-live values are - 
✔️  `aria-live = 'polite' | 'assertive' `
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live#values
https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA19#description

and currently the library have 
❌  `ariaLive: 'polite' | 'alert' = 'polite';`
https://github.com/ng-bootstrap/ng-bootstrap/blob/master/src/toast/toast-config.ts

Fix - aria-live Option `alert` should be replaced with `assertive`.

### Link to minimally-working StackBlitz that reproduces the issue:
https://stackblitz.com/run?file=src%2Fapp%2Ftoast-inline.html

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 18.0.0 

ng-bootstrap: 17.0.1

Bootstrap: 5.3.2
